processes.go

```diff
@@ -57,20 +59,49 @@ func (p ProcessType) Value() (driver.Value, error) {
 
 // Command represents the actual shell command that gets executed for a given
 // ProcessType.
-type Command string
+type Command []string
+
+// ParseCommand parses a string into a Command, taking quotes and other shell
+// words into account.
+func ParseCommand(command string) (Command, error) {
+	return shellwords.Parse(command)
+}
 
 // Scan implements the sql.Scanner interface.
 func (c *Command) Scan(src interface{}) error {
 	if src, ok := src.([]byte); ok {
-		*c = Command(src)
+		command, err := ParseCommand(string(src))
+		if err != nil {
+			return err
+		}
+		*c = command
 	}
 
 	return nil
 }
 
 // Value implements the driver.Value interface.
 func (c Command) Value() (driver.Value, error) {
-	return driver.Value(string(c)), nil
+	// TODO(ejholmes): We really should be storing this as a postgres array,
+	// because stringifying it can cause information to be lost.
+	//
+	// For example, if we have the command:
+	//
+	//	Command{"echo", "hello world"}
+	//
+	// Then stringify it:
+	//
+	//	"echo hello world"
+	//
+	// Then parse it again:
+	//
+	//	Command{"echo", "hello", "world"}
+	return driver.Value(c.String()), nil
```

Comment:
Now that I think about it, this may not be possible without the migration, since parsing -> stringifying -> parsing again isn't pure.